src/KygekDev/IsOP/IsOP.php

```diff
@@ -25,18 +25,19 @@ class IsOP extends PluginBase {
     private const COMMAND_PERMISSION = "isop.command";
 
     public function onCommand(CommandSender $sender, Command $command, string $label, array $args) : bool {
-        if ($command->getName() !== "isop" || !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
+        if ($command->getName() !== "isop" or !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
         
         if (isset($args[0])) {
-            if (in_array(strtolower($args[0]), array_map("strtolower", $this->getOpList()))) {
+            if (in_array(mb_strtolower($args[0]), array_map("strtolower", $this->getOpList()))) {
```

Comment:
`mb_strtolower()` is faster than `strtolower()`

> https://github.com/pmmp/PocketMine-MP/issues/3585#issuecomment-643757656

Reply:
And also better for Unicode strings!